test/functional/signature/cert-chain.bats

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_bundle -n test-bundle -f /test-file "$TEST_NAME"
+
+	key_generation_cmd=$(cat <<-EOM
```

Comment:
I'm not very familiar with these functions, but don't we already have functions to generate these in testlib?
`generate_certificate` and `create_trusted_cacert`, maybe they are not for the same purpose, I'm just pointing them out in case you missed them.

Reply:
I'm using some parameters that are specific for this test. We don't support them in testlib and I don't think it make sense to support, because it's just going to be used in this test.
We are basically checking here if swupd works with this specifics certificates.
But in the key-rotation test you are right. I probably can use the function in testlib. I'll recheck that.

Reply:
make sense, thanks for the explanation, and yeah, I'm really not familiar with those functions related to certificates, so I was shooting in the dark....